src/access_controller/rule.rs

```diff
@@ -1,7 +1,7 @@
 // Copyright (c) 2024 IOTA Stiftung
 // SPDX-License-Identifier: Apache-2.0
 
-use anyhow::Context;
+use anyhow::{Context, Ok};
```

Comment:
Do we need the explicit `Ok::<_, anyhow::Error>(value)` here?

Type inference seems to be fine without it locally for me.

```suggestion
use anyhow::Context;
```